test/crypto/commitment.test.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from "chai";
+import { generatePedersenCommitment, isValidPedersenCommitment } from "../../lib/av_client/crypto/pedersen_commitment";
+
+describe("Pedersen Commitments", () => {
+  it("generate commitment from single message", () => {
```

Comment:
Maybe add some tests that document how errors and edge cases are handled?
What if `messages === []`?
Can `randomizer` be too long or short? Should it be limited to certain chars or are all characters allowed?

Reply:
Good point. I added some more boundary-oriented tests.